### PR TITLE
fix: titlebar color not change

### PR DESCRIPTION
**Pull request by Decodetalkers**
listen to fakewm signal

Log:

### CODE REVIEW

File: src/plugins/kdecorations/deepin-chameleon/chameleonconfig.cpp [hunk 1]
```diff
@@ -983,6 +978,13 @@ void ChameleonConfig::init()
 
     onConfigChanged();
 
+    QDBusConnection::sessionBus().connect(QStringLiteral("com.deepin.wm"),
+                                          QStringLiteral("/com/deepin/wm"),
+                                          QStringLiteral("com.deepin.wm"),
```
Comment (kegechen) [inline src/plugins/kdecorations/deepin-chameleon/chameleonconfig.cpp:983]:
```suggestion
    QDBusConnection::sessionBus().connect(DBUS_DEEPIN_WM_SERVICE,
                                                       DBUS_DEEPIN_WM_OBJ,
                                                       DBUS_DEEPIN_WM_INTF),
```

File: src/plugins/kdecorations/deepin-chameleon/chameleonconfig.cpp [hunk 2]
```diff
@@ -343,13 +343,8 @@ void ChameleonConfig::onWindowShapeChanged(quint32 windowId)
     buildKWinX11ShadowDelay(window);
 }
 
-
```
Comment (kegechen) [inline src/plugins/kdecorations/deepin-chameleon/chameleonconfig.cpp:346]: ?

**Review by kegechen — CHANGES_REQUESTED** [review]

**Review by kegechen — APPROVED** [review]

**Comment by kegechen** [comment]
https://github.com/linuxdeepin/developer-center/issues/5685

**Review by justforlxz — APPROVED** [review]

**Pull request merged**